csrc/moe/moe_ops.cpp

```diff
@@ -1,8 +1,12 @@
+#include "registration.h"
 #include "moe_ops.h"
 
-#include <torch/extension.h>
-
-PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
-  m.def("topk_softmax", &topk_softmax,
-        "Apply topk softmax to the gating outputs.");
+TORCH_LIBRARY_EXPAND(TORCH_EXTENSION_NAME, m) {
+  // Apply topk softmax to the gating outputs.
+  m.def(
+      "topk_softmax(Tensor! topk_weights, Tensor! topk_indices,Tensor "
```

Comment:
I believe `token_expert_indices` is written to here

also nit: whitespace after `,` here and on the next line